lib/mini_racer.rb

```diff
@@ -75,14 +75,12 @@ def write_heap_snapshot(file_or_io)
 
       if String === file_or_io
         f = File.open(file_or_io, "w")
-	implicit = true
+	      implicit = true
```

Comment:
Accidental whitespace change?

Reply:
Ah there must be a tab in there, it looks misaligned in my editor, will fix to use spaces.

Reply:
Fixed.